src/main/java/duke/Event.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    public String eventTime;
+    public LocalDate exactTime;
+
+    public Event(String name, String eventTime) {
+        super(name);
+        this.eventTime = eventTime;
+        try {
+            this.exactTime = LocalDate.parse(eventTime);
+        } catch (DateTimeParseException e) {
+            System.out.println("Wrong input format!");
```

Comment:
The error message can be made clearer here. Perhaps, you can show how the correct input format is really like.